pkgs/applications/audio/spotify-spicified/README.md

```diff
@@ -0,0 +1,87 @@
+# Using Spotify-Spicified / Spicetify-cli's Nix API
```

Comment:
Please move this file to docs as we no longer have docs in the package tree itself.